glean-core/android/src/main/java/mozilla/telemetry/glean/Glean.kt

```diff
@@ -34,16 +36,7 @@ open class GleanInternalAPI internal constructor () {
 
         handle = LibGleanFFI.INSTANCE.glean_initialize(dataDir.path, applicationContext.packageName)
 
-        val e = RustError.ByReference()
-        val pings = listOf("baseline")
-        val ffiPingsList = StringArray(pings.toTypedArray(), "utf-8")
-        bool_metric = LibGleanFFI.INSTANCE.glean_new_boolean_metric(
-                category = "glean",
-                name = "enabled",
-                send_in_pings = ffiPingsList,
-                send_in_pings_len = pings.size,
-                lifetime = Lifetime.Application.ordinal,
-                err = e)
+        initializeCoreMetrics(applicationContext)
```

Comment:
Should we add a comment / bug that initialization happens differently than on legacy?  On legacy we do this sort of "in the middle" and eventually we'll need to call back into Rust to do the last third of initialization.